connector_magento/exception.py

```diff
@@ -7,3 +7,14 @@
 
 class OrderImportRuleRetry(RetryableJobError):
     """ The sale order import will be retried later. """
+
+
+class MagentoError(Exception):
+    """Catch Json Error
+    Attributes:
+        json -- Catching the json error explanation of the failed job
+    """
+
+    def __init__(self, msg, json):
+        super(MagentoError, self).__init__(msg)
```

Comment:
In Python 3 we can simply say super().__init__(msg)

Reply:
done